src/System.Private.Xml/src/System/Xml/Core/XmlTextReaderImpl.cs

```diff
@@ -2579,7 +2574,7 @@ private bool IsResolverNull
         {
             get
             {
-                return _xmlResolver == null || (LocalAppContextSwitches.ProhibitDefaultUrlResolver && !_xmlResolverIsSet);
+                return _xmlResolver == null || !_xmlResolverIsSet;
```

Comment:
Why isn't it just another one in XmlAppContextSwitches?

---
In reply to: [83265236](https://github.com/dotnet/corefx/pull/12609#discussion_r83265236) [](ancestors = 83265236)